sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -913,4 +913,35 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     // Processing any pending promise if not completed.
     await Promise.all(promises);
   }
+
+  /**
+   * Get offering for an changed application Id.
```

Comment:
The description is not right. We are getting the offering change for an application. Application is not changed here.